src/mobile/textarea/test/focusEvent.test.ts

```diff
@@ -1,17 +1,22 @@
 import { expect, fixture } from "@open-wc/testing";
 import { MobileTextArea } from "../index";
 
-describe("Function focus event run successfully", () => {
-  const container = new MobileTextArea({ value: "Orange" });
-  container.addEventListener("focus", (event: any) => {
-    expect(event.detail.value).to.have.equal("Orange");
-  });
+describe("MobileTextArea", () => {
+  describe("focusEvent", () => {
+    it("should be triggered when focusing to the input", async () => {
+      const container = new MobileTextArea({ value: "Orange" });
+      container.addEventListener("focus", (event: any) => {
+        expect(event.detail.value).to.have.equal("Orange");
```

Comment:
Please update as below.

```suggestion
        expect(event.detail.value).to.equal("Orange");
```